ansible_files/roles/aws/prepare_inventory/files/ec2_instances.yml

```diff
@@ -0,0 +1,30 @@
+# files/ec2_instances.yml
```

Comment:
`vars` is more appropriate for such files, but as I said below, I think it should be passed as an extra var